nightfall-optimist/src/services/check-block.mjs

```diff
@@ -52,7 +52,13 @@ async function checkBlockRoot(block) {
       // eslint-disable-next-line no-await-in-loop
       history = await getTreeByLeafCount(block.leafCount);
       logger.debug(`Block has commitments - retrieved history from Timber`);
-      logger.trace(`Timber history was ${JSON.stringify(history, null, 2)}`);
+      logger.trace({
+        msg: 'Timber history was',
+        history,
+      });
+
+      // eslint-disable-next-line no-await-in-loop
+      await new Promise(resolve => setTimeout(resolve, 3000));
```

Comment:
Why do we need this now? And if we do, didn't you create a util? :upside_down_face:

Reply:
I didn't change this point, but you're right, however we need a refactoring task for replacing these instructions with the new method in utils. :D